docs/README.md

```diff
@@ -1,149 +1,77 @@
-
-    ```
-    $ wsk action create hello hello.js
-    ```
-
-3. Then, invoke the action by entering the following commands.
-
-    ```
-    $ wsk action invoke hello --blocking --result
-    ```
-
-    This command outputs:
-
-    ```
-    {
-        "payload": "Hello, World!"
-    }
-    ```
-
-    ```
-    $ wsk action invoke hello --blocking --result --param name Fred
-    ```
-
-    This command outputs:
-
-    ```
-    {
-        "payload": "Hello, Fred!"
-    }
-    ```
-
-You can also use the event-driven capabilities in OpenWhisk to invoke this action in response to events. Follow the [alarm service example](./packages.md#creating-and-using-trigger-feeds) to configure an event source to invoke the `hello` action every time a periodic event is generated.
-
-
-## System details
-
-You can find additional information about OpenWhisk in the following topics:
-
-* [Entity names](./reference.md#openwhisk-entities)
-* [Action semantics](./reference.md#action-semantics)
-* [Limits](./reference.md#system-limits)
-* [REST API](./reference.md#rest-api)
+OpenWhisk is an [Apache Incubator Project](https://incubator.apache.org/projects/openwhisk.html). It is an open source implementation of a distributed, event-driven compute service. You can run it on your own hardware on-prem, or in the cloud. When running in the cloud you could use PaaS version of the OpenWhisk provided by IBM Bluemix cloud, or you can provision it yourself into Bluemix IaaS, or other IaaS clouds, such as Amazon AWS, Microsoft Azure, Google GCP, etc. 
+
+OpenWhisk runs application logic in response to events or direct invocations from web or mobile apps over HTTP. Events can be provided from Bluemix services like Cloudant and from external sources. Developers can focus on writing application logic, and creating actions that are executed on demand. The benefits of this new paradigm are that you do not longer need to explicitly provision servers and worry about auto-scaling, high availability, updates, maintenance and pay for hours of processor time when your server is running, but not serving requests. Your code gets called whenever there is an HTTP call, database state change, or other type of event that triggers the execution of your code. You get billed by request or per millisecond of execution time, not per hour of JVM regardless whether that VM was doing useful work or not.
+
+This programming model is perfect match for microservices, mobile, IoT and many other apps – you get inherent auto-scaling and load balancing out of the box without having to manually configure clusters, load balancers, http plugins, etc. If you happen to run on IBM Bluemix cloud, you also get zero administration benefit. All you need to do is to provide the code you want to execute and give it to your cloud vendor. The rest is “magic”. Good introduction into the serverless programming model is available on [Martin Fowler's blog](https://martinfowler.com/articles/serverless.html).
```

Comment:
model is perfect ->  model is a perfect 
you also get zero administration benefit. -> reword, reads funny.